docs/plugin/index.md

```diff
@@ -1,8 +1,15 @@
-# Botkube plugin
+---
```

Comment:
It would be good to mention the repo template somewhere here, and probably link to it in the "Custom executor/source" tutorials as well 🤔

Reply:
will do in a follow-up